terraform/modules/database/security_groups.tf

```diff
@@ -0,0 +1,16 @@
+# Security groups for the database
+
+resource "aws_security_group" "rds-postgres-sg" {
+  name   = "${var.environment}-rds-sg"
+  vpc_id = var.vpc_id
+}
+
+# Inbound Security rule
+resource "aws_security_group_rule" "rds-inbound" {
+  type              = "ingress"
+  from_port         = 5432
+  to_port           = 5432
+  protocol          = "tcp"
+  cidr_blocks       = ["0.0.0.0/0"]
```

Comment:
This means that we allow anyone to access our db.
Is that intentional?

I'd prefer to lock it down to eg a certain SG via `source_ security_group_id`

Reply:
That certainly isnt what i meant. It does have a secruity group below, so maybe it is on that. 

I'll https://github.com/openclimatefix/nowcasting_infrastructure/issues/25